src/Examples/Program.cs

```diff
@@ -1,10 +1,12 @@
 // Copyright (c) .NET Foundation and Contributors.  All Rights Reserved.  See LICENSE in the project root for license information.
 
+using System.Threading.Tasks;
+
 namespace TorchSharp.Examples
 {
     public static class Program
     {
-        public static void Main(string[] args)
+        public static async Task Main(string[] args)
```

Comment:
It seems unnecessary to make this asynchronous. The program is a console app, and you have to await the result, anyway.

Reply:
Done.